temporalio/bridge/src/worker.rs

```diff
@@ -0,0 +1,174 @@
+use prost::Message;
+use pyo3::exceptions::{PyRuntimeError, PyValueError};
+use pyo3::prelude::*;
+use std::time::Duration;
+use temporal_sdk_core_api::Worker;
+
+use crate::client;
+
+pyo3::create_exception!(
+    temporal_sdk_bridge,
+    PollShutdownError,
+    pyo3::exceptions::PyException
+);
+
+#[pyclass]
+pub struct WorkerRef {
+    worker: std::sync::Arc<temporal_sdk_core::Worker>,
```

Comment:
Yeah, I often only import for required trait inference purposes on some of these smaller files. I have traditionally preferred such consistency it over the inconsistency of half of the time qualifying names and half of the time not. Is there a recommended consistent approach we can take (or a cargo fmt setting)?

Reply:
For consistency I would say import unless using the direct name is very ambiguous. Like `spawn` for example, you probably want some extra qualifier there. `Arc` and `WorkerConfig`, etc, are rather clear

Reply:
The problem with "probably" and "rather clear" is that is subjective to the writer. I was hoping there was a way we could have it not be per author's subjective feelings. Python suffers this same problem (luckily Go, Java, etc do not).

Reply:
Sure, it's subjective, but I'd say like 90% of the time I am importing something directly, and it's worth it for reducing noise in the code vs 100% full paths. I'd say it's worse in something like Java since you don't even get to decide if you want to be more clear or not.

Reply:
I have moved all that I reasonably can to imports and relative names.